website/cue/reference/releases/0.43.0.cue

```diff
@@ -184,7 +184,7 @@ releases: "0.43.0": {
 
 					1. The message must be added in `.doc` and have `.doc_as_upsert` to true.
 					2. `id_key` must be set, and the `encoding` field should specify `doc` and `doc_as_upsert` as values
-				"""
+					"""
```

Comment:
I think you made the wrong indentation change here. The `"""`s were indented correctly. The text was indented one level too deep. See the entry above this one.

Reply:
I checked that it renders correctly both ways. Changed to https://github.com/vectordotdev/vector/pull/21906/commits/8e36c6eda15e2c7a6742e30010f2ab91a9c94479 for consistency.